src/test/java/org/junitpioneer/platform/testkit/engine/PioneerTestKit.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.platform.testkit.engine;
+
+public class PioneerTestKit {
+
+	/**
+	 * Returns the execution results of the given test class.
+	 *
+	 * @param testClass Name of the test class, the results should be returned
+	 * @return The execution results
+	 */
+	public static PioneerEngineExecutionResults getExecutionResults(Class<?> testClass) {
+		PioneerEngineExecutionResults results = new PioneerEngineExecutionResults(testClass);
+
+		return results;
```

Comment:
You could inline this (i.e.: `return new Pioneer...`).

Reply:
See above